src/generic_core/metrics.ts

```diff
@@ -25,9 +30,16 @@ export class Metrics {
       num_cohorts: 64, prob_p: 0.5, prob_q: 0.75, prob_f: 0.5,
       flag_oneprr: true
     };
+    var stringMetric = {
+      type: 'string', num_bloombits: 8, num_hashes: 2,
```

Comment:
@willscott @mollyling - can you folks double check these values here?  We recently had an issue regarding an incorrect setting for the bloombits (#1644), want to make sure we won't have similar issues again